dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -170,6 +170,7 @@ def show
       if @script_level.lesson.rubric && view_as_other
         viewing_user_level = @view_as_user.user_levels.find_by(script: @script_level.script, level: @level)
         @rubric_data[:studentLevelInfo] = {
+          id: @view_as_user.id,
```

Comment:
nit: can we call this `user_id`? otherwise it is a bit ambiguous later on whether it is the user id or level id